widget/embedded/src/components/HeaderButtons/HomeButtons.tsx

```diff
@@ -53,12 +59,18 @@ export function HomeButtons(props: HomeButtonsPropTypes) {
           </HeaderButton>
         </Popover>
       </Tooltip>
-      <Tooltip side="top" content={i18n.t('Settings')}>
+      <Tooltip
+        container={document.getElementById('swap-box') as HTMLElement}
+        side="top"
+        content={i18n.t('Settings')}>
         <HeaderButton size="small" variant="ghost" onClick={onClickSettings}>
           <SettingsIcon size={18} color="black" />
         </HeaderButton>
       </Tooltip>
-      <Tooltip side="top" content={i18n.t('Transactions History')}>
+      <Tooltip
+        container={document.getElementById('swap-box') as HTMLElement}
+        side="top"
+        content={i18n.t('Transactions History')}>
         <HeaderButton size="small" variant="ghost" onClick={onClickHistory}>
           <TransactionIcon size={18} color="black" />
         </HeaderButton>
```

Comment:
you can create a variable like tooltip container and remove duplicate code:
const tooltipContainer = document.getElementById('swap-box') as HTMLElement